diff --git a/src/PackagesBin.cpp b/src/PackagesBin.cpp
--- a/src/PackagesBin.cpp
+++ b/src/PackagesBin.cpp
@@ -518,8 +518,8 @@ PackagesBin::findAllValueOffsets(BinaryReader::BufferedSlice& reader, std::array<size_t, 5>& offsets, int state)
     const std::vector<std::tuple<uint32_t, uint32_t, std::string>> searchValueRangesWarframe = {
         {45, 60, "ReferenceCount"},
         {75000, 90000, "ComFlagsBufLen"},
-        {210000, 240000, "ComSizeBufferLen"},
-        {18000000, 21000000, "ComZBufferLen"},
+        {210000, 300000, "ComSizeBufferLen"},
+        {18000000, 26000000, "ComZBufferLen"},
         {9, 16, "PkgNameLen"},
         {80000, 1300000, "DictSize"},
         {400000, 600000, "EntityCount"},
